ActionGame/ActionGame/Script/sc_Result.cpp

```diff
@@ -0,0 +1,31 @@
+
+#include "Header/Common.h"
+
+Scene_Result::Scene_Result() {
+
+}
+
+Scene_Result::~Scene_Result() {
+
+}
+
+void Scene_Result::SceneProcessing() {
+	GameProcessing();
+	DrawProcessing();
+}
+
+void Scene_Result::GameProcessing() {
+	if ( fadeMode == Mode_FadeNone ) {
+		if ( CheckHitKey( KEY_INPUT_3 ) ) {
+			fadeMode = Mode_FadeOut;
+		}
+	}
```

Comment:
全く問題ですが、
```suggestion
	if ( fadeMode != Mode_FadeNone ) return;
	
	if ( CheckHitKey( KEY_INPUT_3 ) ) {
		fadeMode = Mode_FadeOut;
	}
```
のようにするとネストが減って読みやすくなりますよ。

Reply:
@taishi-matsumura 
9f4ec70 で修正しました。

ついつい深くしがちなので気を付けます。